config/log4j.properties

```diff
@@ -21,70 +21,73 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
 
-log4j.appender.kafkaAppender=com.automq.shell.log.S3RollingFileAppender
+log4j.logger.com.automq.log.uploader.S3RollingFileAppender=INFO, stdout
+log4j.additivity.com.automq.log.uploader.S3RollingFileAppender=false
+
+log4j.appender.kafkaAppender=com.automq.log.uploader.S3RollingFileAppender
 log4j.appender.kafkaAppender.MaxFileSize=100MB
 log4j.appender.kafkaAppender.MaxBackupIndex=14
 log4j.appender.kafkaAppender.File=${kafka.logs.dir}/server.log
 log4j.appender.kafkaAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.kafkaAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
 
-log4j.appender.stateChangeAppender=com.automq.shell.log.S3RollingFileAppender
+log4j.appender.stateChangeAppender=com.automq.log.uploader.S3RollingFileAppender
 log4j.appender.stateChangeAppender.MaxFileSize=10MB
 log4j.appender.stateChangeAppender.MaxBackupIndex=11
 log4j.appender.stateChangeAppender.File=${kafka.logs.dir}/state-change.log
 log4j.appender.stateChangeAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.stateChangeAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
 
-log4j.appender.requestAppender=com.automq.shell.log.S3RollingFileAppender
+log4j.appender.requestAppender=com.automq.log.uploader.S3RollingFileAppender
 log4j.appender.requestAppender.MaxFileSize=10MB
 log4j.appender.requestAppender.MaxBackupIndex=11
 log4j.appender.requestAppender.File=${kafka.logs.dir}/kafka-request.log
 log4j.appender.requestAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.requestAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
 
-log4j.appender.cleanerAppender=com.automq.shell.log.S3RollingFileAppender
+log4j.appender.cleanerAppender=com.automq.log.uploader.S3RollingFileAppender
```

Comment:
It's a little bit strange to have S3RollingFileAppender in a uploader package. Maybe it should in com.automq.log package